web/content/docs/benchmarks/hydro-mechanics/PermeabilityMohrCoulombFailureIndexModel.pandoc

```diff
@@ -0,0 +1,24 @@
++++
+date = "2020-06-11"
+title = "Permeability in EDZ"
+weight = 151
+project = "HydroMechanics/FailureIndexDependentPermeability/quad_with_half_hole.prj"
+author = "Wenqing Wang"
+
+[menu]
+  [menu.benchmarks]
+    parent = "hydro-mechanics"
+
++++
+
+{{< data-link >}}
+
+For the description of the benchmark and its related permeability model,
+  please read [the PDF](../PermeabilityMohrCoulombFailureIndexModel.pdf).
+
+For a full document of the failure index dependent
+ permeability model, benchmark and its the implementation in ogs6, etc.,
```

Comment:
@OlafKolditz 
 The link to the document on Overleaf was add. Please see [Preview](https://deploy-preview-2995--ogs.netlify.app/docs/benchmarks/hydro-mechanics/permeabilitymohrcoulombfailureindexmodel/)